components/movementsensor/imuwit/imu.go

```diff
@@ -1,15 +1,32 @@
 // Package imuwit implements wit imus.
-// Tested on the HWT901B and BWT901CL models. Other WT901-based models may work too.
 package imuwit
 
+/*
+Sensor Manufacturer:  		Wit-motion
+Supported Sensor Models: 	HWT901B, BWT901, BWT61CL
+Supported OS: Linux Debian GNU/Linux 11 (bullseye)
+Tested Sensor Models and User Manuals:
+
+	BWT61CL: https://drive.google.com/file/d/1cUTginKXArkHvwPB4LdqojG-ixm7PXCQ/view
+	BWT901:	https://drive.google.com/file/d/18bScCGO5vVZYcEeNKjXNtjnT8OVlrHGI/view
+	HWT901B TTL: https://drive.google.com/file/d/10HW4MhvhJs4RP0ko7w2nnzwmzsFCKPs6/view
+
+This driver will connect to all the sensors using a usb connection given as a serial path
```

Comment:
"all the sensors" implies that if you plug in multiple sensors, one component will interact with them all. I'd say it connects to "a sensor" or "the sensor" or perhaps "its sensor" instead.